src/components/TopicTelephoneInfos/TopicTelephoneInfos.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../globals.scss';
+
+.wkp-tel-infos {
+  z-index: 1;
+  background-color: white;
+  border: 2px solid #666;
+  color: $brand-secondary;
+  padding: 10px;
+  width: 200px;
+  position: absolute;
+  top: 90px;
+  left: 50%;
+  transform: translate(-50%, 0);
+}
+
+.tm-w-m,
+.tm-w-s,
+.tm-w-xs {
+  .wkp-tel-infos {
+    top: 60px;
+  }
+}
```

Comment:
Put them in trafimage-maps.scss with the others

Reply:
Didn't saw that we locate all screen size specific style in TrafimageMaps.scss now.
I updated it :)